src/ResourceManager/Security/Commands.Security/AzureRM.Security.psd1

```diff
@@ -3,7 +3,7 @@
 #
 # Generated by: Microsoft Corporation
 #
-# Generated on: 9/13/2018
+# Generated on: 12/11/2018
```

Comment:
ANy changes should be in Az.Security, AzureRM.Security is removed in the master branch